File: front-end/Nova/nova-core/plugins/CommandNotFoundPlugin.ts
import NovaPlugin from '../abstractPlugin';

export default class CommandNotFoundPlugin extends NovaPlugin {

    executeSecondaryCommnand(command: string): string {
        return 'Can you rephrase what you said?';
    }

    getKeywords(): string[] {
        return [];
    }

    execute(command: string): string {
        return 'I\'m sorry, I don\'t understand.';
    }
}

File: nova-core/plugins/command_not_found_plugin.py
from core.abstract_plugin import NovaPlugin

class CommandNotFoundPlugin(NovaPlugin):

    def get_keywords(self) -> list:
        return []

    def execute(self, command: str) -> str:
        return 'I\'m sorry, I don\'t understand'

    def help_command(self, command: str) -> str:
        return 'This command is used by the core to respond to users when there is an error'